frontend-v2/src/features/data/MapDosing.tsx

```diff
@@ -81,7 +82,7 @@ const MapDosing: FC<IMapDosing> = ({ state, firstTime }: IMapDosing) => {
     />
   ) : (
     <CreateDosingProtocols
-      administrationIdField={"Group"}
+      administrationIdField={administrationIdField || "Group ID"}
```

Comment:
If we upload a CSV that only has observations, no amounts or administration IDs, this should fall back to mapping dosing compartments to each of the groups that were created during the Stratification step.